### ISSUE TITLE
Uncaught TypeError: Cannot read property 'constructor' of undefined - lottie.js:11823

### ISSUE THREAD

**Issue by tomgreer** [airbnb/lottie-web#1483]
<img width="1100" alt="screenshot-lottie-error" src="https://user-images.githubusercontent.com/21129034/51987074-a33a7d80-246f-11e9-9653-b5905151404e.png">

**Tell us about your environment**
json created in After Effects 2019 on macOS running High Sierra
lottie.js using latest version from CDN (5.4.3)

* **Browser and Browser Version:**
Same results in Chrome 71.0.3578.98 and Firefox 64.0.2 

* **After Effects Version:**
16.0.1 (Build 48)

**What did you do? Please explain the steps you took before you encountered the problem.**
Created HTML page per documentation. 
<div id="trg-logo"></div>
<script src="https://cdnjs.cloudflare.com/ajax/libs/bodymovin/5.4.3/lottie.js" type="text/javascript"></script>
<script>
var animation = lottie.loadAnimation({
  container: document.getElementById('trg-logo'), // Required
  path: 'animated-logo.json', // Required
  renderer: 'svg',
  loop: true,
  autoplay: true,
})
</script>

See page here: https://trgwebdev.com/trgwebdev/logo.html

**What did you expect to happen?**
Animated logo to be displayed

**What actually happened? Please include as much _relevant_ detail as possible.**
Webpage displays unrecognizable static image

**Please provide a download link to the After Effects file that demonstrates the problem.**
https://trgwebdev.com/trgwebdev/animated-logo.json

**Comment by bodymovin** [comment]
hi, can you share the .aep?

**Comment by tomgreer** [comment]
Here is a link: https://drive.google.com/open?id=1PsBAelhGGN9NL-PDOmWdZwwlbKTb2Dae

**Comment by tomgreer** [comment]
Thanks.

   <https://trgwebdesigns.com/>	 	
Tom Greer
President/Owner, TRG Web Designs
tom.greer@trgwebdesigns.com
Office: 317-520-2712
Mobile: 317-800-4455
www.trgwebdesigns.com <https://trgwebdesigns.com/> <https://www.facebook.com/trgwebsitedesigns> <https://twitter.com/trgwebdesigns> <https://www.instagram.com/trgweb/> <https://www.pinterest.com/greer0898/> <https://www.linkedin.com/company/10312206> <https://www.youtube.com/channel/UCZWQZtqIoaV-aqfWjzKwmmA>

**Comment by JasonSchrode** [comment]
Hey there,

I had a similar issue as Tom Greer above. I was trying to test a very simple animation using the plugin, and I kept getting an error that kicked back:
"Uncaught TypeError: Cannot read property 'loadAnimation' of undefined".

I was using After Effects CC 2019 v16.0.1 on Windows 10 Pro, and using Sublime Text 3.0 build-3143. The video was 0:02s long, 1920x1080px showing a simple blue square that rotates 90 degrees with easing. There are no additional effects or plugins being used in this project. My file structure has been confirmed to be correctly working with other files so the issue most likely resides somewhere other than the project folder structure.

Any insight into why this set up is not working would be greatly appreciated. I tried following the Github readme as closely as possible, and I've still not been able to run the plugin properly.

Thanks for any help.



**Here's my HTML**

<!DOCTYPE html>
<html>
<head>
	<meta charset="UTF-8" />
	<title>BodyMovin Test</title>
	<link rel="stylesheet" type="text/css" href="assets/css/main.css">
</head>

	<body>
		<div id="bm"></div>

		<script src="assets/js/main.js"></script>
		<script src="https://cdnjs.cloudflare.com/ajax/libs/bodymovin/5.4.4/lottie.js"></script>
	</body>

</html>





**Here's my CSS**

@charset "UTF-8";

* { box-sizing: border-box;}

html,
body {
  background-color: #444444;
  width: 100%;
  height: 100%;
  margin: 0;
  overflow: none;
}

#bm {
  width: 100%;
  height: 100%;
  border: 4px solid blue;
}






**Here's my JS**

var lottie;

var element = document.getElementById('bm');

lottie.loadAnimation({
	container: element, // the dom element that will contain the animation
	renderer: 'svg',
	loop: true,
	autoplay: true,
	path: 'data.json' // the path to the animation json
})


Again, the error kicks back:
"Uncaught TypeError: Cannot read property 'loadAnimation' of undefined"
referenced at line 6 in the editor, which contains the loadAnimation appendage to lottie. 

Thanks for any help, it's greatly appreciated, 

jason

**Comment by bodymovin** [comment]
hi @JasonSchrode it's probably the order of your scripts. You're trying to use lottie before it's been loaded in the page.
Try inverting them.

**Comment by JasonSchrode** [comment]
Hey @bodymovin thanks for the quick response. I was able to successfully load the animation into my browser after inverting the scripts as you suggested, as well as exporting the data from After Effects as a JS file (using "Standalone" render setting) as opposed to a JSON (using "Glyphs" setting). However, the auto-looping functionality is not working, and I'm getting an error saying:

**VM305:1 Uncaught SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at Object.loadAnimation (data.js:1)
    at main.js:6
loadAnimation @ data.js:1
(anonymous) @ main.js:6**



The snippet of code it seems to be referencing in data.js is this:

**function loadAnimation(t){return standalone===!0&&(t.animationData=JSON.parse(animationData)),animationManager.loadAnimation(t)}**

P.S. I was receiving errors before regarding CORS restrictions, but the issue was resolved after loading the "lottie.js" script before "data.js" script. I'm not sure if the new error is related to CORS protocol, but I thought it may be useful background information. 

If you have any idea as to why this may be occurring, that would be awesome. Glad it's loading properly first though, that was a huge step. 

Thanks,

Jason
